Library/Homebrew/formula_installer.rb

```diff
@@ -101,6 +100,7 @@ def pour_bottle?(install_bottle_options = { warn: false })
       return false
     end
 
+    bottle = formula.bottle_specification
```

Comment:
Why is this needed below, out of interest, and `bottle_specification` rather than `bottle`?

Reply:
`formula.bottle` returns `nil` when `compatible_cellar?` is false.
See https://github.com/Homebrew/brew/blob/master/Library/Homebrew/formula.rb#L330
and https://github.com/Homebrew/brew/blob/master/Library/Homebrew/software_spec.rb#L93

Rather than `formula.bottle_specification.compatible_cellar?` we could use `formula.bottled?`. Do you have a preference?

Reply:
`formula.bottled?` would be preferable, yeh.

Reply:
Done.